.eslintrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     "airbnb",
     "plugin:react-hooks/recommended",
     "plugin:prettier/recommended",
+    "plugin:@kiwicom/orbit-internal/recommended",
```

Comment:
I would not add for the internal plugin now. We have been talking with @silvenon that we might merge these two plugins and we could use `eslint-plugin-orbit-components` internally. We'll discuss it more tomorrow.